plugins/BEdita/Core/src/Model/Entity/ObjectType.php

```diff
@@ -315,4 +385,25 @@ protected function _getSchema()
 
         return compact('properties', 'required');
     }
+
+    /**
+     * Add internal associations as properties
+     *
+     * @return array
+     */
+    protected function associationProperties(): array
+    {
+        $associations = ['Tags', 'Categories', 'DateRanges'];
```

Comment:
Maybe define these as a class property? This way it can be overridden/changed more easily.

Reply:
Done as class constant for now. 
It's not something we need to change/override at the moment (i think).
Better late than never :wink: